ckanext/datarequests/plugin.py

```diff
@@ -17,41 +17,15 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with CKAN Data Requests Extension. If not, see <http://www.gnu.org/licenses/>.
 
-import ckan.lib.helpers as h
-import ckan.plugins as p
-import ckan.plugins.toolkit as tk
-import auth
```

Comment:
is the from ckanext.datarequests import better for reducing miss filename pickups?

Reply:
I'm not an expert on Python module loading, but it is more readable, anyway, making it immediately clear whether something is a sibling or a built-in module.